### ISSUE TITLE
Community Chart Suggestion: Moodle

### ISSUE THREAD

**Issue by pabloeisenhut** [truenas/apps#206]
https://moodle.org/

**Comment by RemyDG0** [comment]
I agree that Moodle is a very nice learning platform and would be a nice addition as an app on TrueNas Scale

**Comment by stavros-k** [comment]
I dont see a moodle docker image available, Even the one shared by @RemyDG0 is not published, but you have to build it yourself. (Besides that is 11years old)
So for now is not going to happen
Thanks
